device_specific/flash_the_m5stack_core_basic

```diff
@@ -15,11 +15,11 @@ working_directory="${HOME}/Downloads/diy_jade"
 temp_directory="${working_directory}/temp"
 trap cleanup EXIT
 
-jade_git_tag="1.0.26"
++jade_git_tag="1.0.29"
```

Comment:
Can you please remove the extra `+` here and on line 18/19 of the other scripts?

Reply:
sorry, please check again